app/api/views.py

```diff
@@ -2,22 +2,17 @@
 from django.http import JsonResponse
 from django.utils.encoding import force_text
 from django.core.serializers.json import DjangoJSONEncoder
+import util
```

Comment:
Please always use absolute imports: `from api import util`. Otherwise, results may be different if it happens that there's a package called `util` somewhere in the system.

(Granted, the `api` module is not the best name choice either, but not your fault :) )